packages/server/test/support/fixtures/projects/ts-proj/cypress.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
~Instead of adding empty support files for this project, let's configure it with `"supportFile": false` and then we can leave them out.~

Reply:
Sorry, ignore the previous comment. I guess we should have them since we are testing `.ts` files, but I feel like they should do something. Same with the plugins file.

Reply:
Copied examples from Cypress website and used them.